src/index.css

```diff
@@ -25,3 +25,19 @@ code {
 .active-link {
   font-weight: bold;
 }
+
+.soon {
+  background-color: green;
+}
+
+.kindOfSoon {
+  background-color: orange;
+}
+
+.notSoSoon {
+  background-color: red;
+}
+
+.inactive {
+  background-color: gray;
+}
```

Comment:
It looks like you have 2 ways of styling! Are you planning on using this later on in the future?

Reply:
I think Isa forgot to delete the styles, we implemented with CSS first but later changed it. We should take those out  👍🏻 Thanks for noticing, Sandy!